src/assets/Generator.Shared/TypeFormatters.cs

```diff
@@ -26,8 +26,8 @@ internal class TypeFormatters
         {
             "D" => value.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture),
             "U" => value.ToUnixTimeSeconds().ToString(CultureInfo.InvariantCulture),
-            "O" when value.Offset == TimeSpan.Zero => value.ToString(RoundtripZFormat, CultureInfo.InvariantCulture),
-            "o" when value.Offset == TimeSpan.Zero => value.ToString(RoundtripZFormat, CultureInfo.InvariantCulture),
+            "O" => value.ToUniversalTime().ToString(RoundtripZFormat, CultureInfo.InvariantCulture),
```

Comment:
Can we add unit tests here to verify that if I deserialize an object, store it some then reserialize it we don't have any data loss when it comes to timezone?